easybuild/easyconfigs/a/at-spi2-core/at-spi2-core-2.36.0-GCCcore-9.3.0.eb

```diff
@@ -29,6 +29,8 @@ dependencies = [
     ('X11', '20200222'),
 ]
 
+# Hard disable Dbus broker detection
+preconfigopts = "sed -i s/dbus_broker.found\(\)/false/ ../*/bus/meson.build &&"
```

Comment:
This is failing python linting check:
```
./easybuild/easyconfigs/a/at-spi2-core/at-spi2-core-2.36.0-GCCcore-9.3.0.eb:33:44: W605 invalid escape sequence '\('
```
Anyone know how to fix that?

Reply:
fixed in ae49fecfae479ae64dc19775a1878026b98c5920